src/screens/profile/screen.tsx

```diff
@@ -2,29 +2,36 @@ import { Box, Heading, Link, SimpleGrid, Text } from '@chakra-ui/core';
 import { TopNav } from '../../components/top-nav';
 import { PageFooter } from '../../components/page-footer';
 import { LazyImage } from '../../components/lazy-image';
+import markdownStyles from './markdown-styles.module.css'
+import { memberProfileType } from '../../../pages/profile/[name]';
+
+
+export function ProfileScreen({ profile }: memberProfileType) {
+  const { username, imageUrl, designation, company, content } = profile;
 
-export function ProfileScreen() {
   return (
     <>
       <TopNav />
 
-      <Link color='gray.400'>&larr; Back to Home</Link>
+      <Link color='gray.400' href='/'>&larr; Back to Home</Link>
 
       <SimpleGrid gridTemplateColumns={ ['1fr', '1fr', '200px 1fr', '200px 1fr'] } mt='40px'>
-        <LazyImage width={ '200px' } height='auto' skeletonHeight='200px' src='/avatars/kamran.jpeg' rounded='10px' />
-        <Box ml={ [0, 0, '30px', '30px'] }>
+      <LazyImage width={'200px'} height='auto' skeletonHeight='200px' src={imageUrl} rounded='10px' />
+        <Box ml={ [0, 0, '30px', '30px'] }  mt={ ['20px', 0] }>
           <Box mb='30px'>
-            <Heading color='gray.100' fontSize='45px' lineHeight='45px' mb='12px'>Kamran Ahmed</Heading>
-            <Text color='gray.400' fontSize='17px'>Engineering Manager at Tradeling</Text>
+            <Heading color='gray.100' fontSize='45px' lineHeight='45px' mb='12px'>{username}</Heading>
```

Comment:
I think instead of username we should use title here for display names.

`<Heading color='gray.100' fontSize='45px' lineHeight='45px' mb='12px'>{title}</Heading>`

Reply:
@AdnanLatif you are right will fix it

Reply:
Thanks.